src/main/kotlin/no/nav/syfo/application/exception/LegeNotFoundException.kt

```diff
@@ -0,0 +1,5 @@
+package no.nav.syfo.application.exception
+
+class LegeNotFoundException(
+    message: String,
```

Comment:
Could we add default value? In case it's kinda general thing we don't have more explanation for? Or  most used error message?